main.nf

```diff
@@ -638,46 +637,35 @@ FINAL_CLUSTERS.into{
 
 // Find out what resolutions are represented by the marker files
 
-process mark_marker_resolutions {
+process mark_marker_param {
 
     executor 'local'
     
     input:
-        file markersFile from SCANPY_CLUSTER_MARKERS
+        file markersFile from SCANPY_MARKERS
 
     output:
-        set stdout, file (markersFile) into CLUSTER_MARKERS_BY_RESOLUTION 
+        set stdout, file ('cluster_markers.tsv') optional true into CLUSTER_MARKERS_BY_RESOLUTION 
```

Comment:
what happens downstream if no output is produced?

Reply:
The summary step would likely fail, triggering an error at the top level which would need to be investigated.